common/src/main/java/com/trainlab/repository/UserRepository.java

```diff
@@ -6,10 +6,9 @@
 import java.util.List;
 import java.util.Optional;
 
-//@Cacheable("users")
 public interface UserRepository extends JpaRepository<User, Long> {
-    Optional<User> findByAuthenticationInfoEmail(String email);
+    Optional<User> findByAuthenticationInfoEmailAndIsDeletedFalse(String email);
 
     List<User> findAllByIsDeletedFalseOrderById();
-
+    Optional<User> findByIsDeletedFalseAndId(Long id);
```

Comment:
Аргументы которые нужно указывать лучше ставить в начале в названии

```suggestion
    Optional<User> findByIdAndIsDeletedFalse(Long id);
```